Sources/App/Resources/en.lproj/Localizable.strings

```diff
@@ -759,4 +759,44 @@ Home Assistant is free and open source home automation software with a focus on
 "widgets.open_page.description" = "Open a frontend page in Home Assistant.";
 "widgets.open_page.not_configured" = "No Pages Available";
 "widgets.open_page.title" = "Open Page";
-"yes_label" = "Yes";
\ No newline at end of file
+"yes_label" = "Yes";
+"carplay.labels.buttons" = "Buttons";
+"carplay.labels.covers" = "Covers";
+"carplay.labels.input_booleans" = "Input Booleans";
+"carplay.labels.input_buttons" = "Input Buttons";
+"carplay.labels.lights" = "Lights";
+"carplay.labels.locks" = "Locks";
+"carplay.labels.scenes" = "Scenes";
+"carplay.labels.scripts" = "Scripts";
+"carplay.labels.switches" = "Switches";
+"carplay.labels.servers" = "Servers";
+"carplay.labels.empty_domain_list" = "No domains available";
+"carplay.labels.no_servers_available" = "No servers available. Add a server at home assistant Companion App.";
+"carplay.labels.already_added_server" = "Already added";
+"state.auto" = "Auto";
+"state.cleaning" = "Cleaning";
+"state.closed" = "Closed";
+"state.closing" = "Closing";
+"state.cool" = "Cool";
+"state.docked" = "Docked";
+"state.dry" = "Dry";
+"state.error" = "Error";
+"state.fan_only" = "Fan Only";
+"state.heat_cool" = "Heat Cool";
+"state.heat" = "Heat";
+"state.idle" = "Idle";
+"state.jammed" = "Jammed";
+"state.locked" = "Locked";
+"state.locking"= "Locking";
+"state.off" = "Off";
+"state.on" = "On";
+"state.open" = "Open";
+"state.opening" = "Opening";
+"state.paused" = "Paused";
+"state.returning" = "Returning";
+"state.unavailable" = "Unavailable";
+"state.unknown" = "Unknown";
+"state.unlocked" = "Unlocked";
+"state.unlocking" = "Unlocking";
+"state.recording" = "Recording";
+"state.streaming" = "Streaming";
```

Comment:
We should download the state translations from core, instead of having them here.

We can use the websocket command `frontend/get_translations`

<img width="796" alt="CleanShot 2023-06-30 at 17 58 02@2x" src="https://github.com/home-assistant/iOS/assets/5662298/eb9754b9-bef9-4069-9edd-173578fa2c5c">

Reply:
the translation loading script (https://github.com/home-assistant/iOS/blob/master/fastlane/Fastfile#L253-L254) can handle loading translations from the frontend repo at build time, avoiding dynamic problems. the translations need to be tagged in the frontend repo with ios so that it pulls them down.

Reply:
These translations are not in the frontend, but in core. The frontend loads them over the websocket.

Reply:
@zacwest and @bramkragten What do I have to do to get these translations at build time?

Reply:
Done yesterday, now we just need to check if any translation is missing and tag on lokalise. In the project you can see those already downloaded on CoreStrings and FrontendStrings